plutus-tx-plugin/plutus-tx-plugin.cabal

```diff
@@ -81,6 +81,22 @@ library
     else
         build-depends: ghc >=8.10 && <9
 
+executable gen-plugin-opts-doc
+    import: lang
+    main-is: GeneratePluginOptionsDoc.hs
+    hs-source-dirs:
+        app
+    ghc-options: -threaded -rtsopts -with-rtsopts=-N
```

Comment:
Does this SMP parallelism add anything?

Reply:
I think this is actually good practice for all executables, we should probably do it everywhere. One for the style guide, perhaps.

Reply:
It makes your program use multiple capabilities at all (I think)!. Otherwise it's easy to accidentally make things single-threaded. Which is quite annoying to track down, and it mostly doesn't hurt when you don't need it, which is why it's sensible to have it on by default.

Reply:
I would say that when creating an executable, the developer would most likely be aware if utilizing a multithread library underneath, so underutilizing SMP (aside the parallel gc) must be quite rare (IMO)